# Msg containing highlevel information about the car

float32 speed
float32 distance  # Distance driven along path

simulation_groundtruth/Section current_section
